<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QDialog" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员模式</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>160</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="AddverticalLayout">
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="AddName">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入新用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="AddPassword">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入新用户密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="AddType">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入新用户类型</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>160</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="DeleteverticalLayout">
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="DeleteName">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入要删除的用户名</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>160</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="EditverticalLayout">
    <item>
     <widget class="QPushButton" name="EditButton">
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="EditedName">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入要修改的用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditName">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入新用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditPassword">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入新密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EditType">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入新类型</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/Logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
